src/mathjs-lang.js

```diff
@@ -5,7 +5,7 @@
  *
  * @param {Object} math A mathjs instance
```

Comment:
Can you update the JSDoc so it matches the new `getMath` and `getScope` arguments?

Reply:
yes of curse, thanks!